src/language-server/client.ts

```diff
@@ -170,12 +171,9 @@ export class LanguageServerClient extends EventEmitter {
   }
 
   private async getInitializationOptions(): Promise<any> {
+    const branch = microPythonConfig.stubs;
     const typeshed = await retryAsyncLoad(() => {
-      switch (this.locale) {
```

Comment:
Need to review whether this is still bundle splitting correctly. I think it's cleverer than I give it credit for though.